pkg/querier/queryrange/codec.go

```diff
@@ -109,7 +109,7 @@ func (r *LokiInstantRequest) GetStart() time.Time {
 
 func (r *LokiInstantRequest) WithStartEnd(s time.Time, _ time.Time) queryrangebase.Request {
 	clone := *r
-	clone.TimeTs = s
+	clone.TimeTs = time.Unix(0, s.UnixNano())
```

Comment:
This might be changing some semantics; can you explain what the purpose of this change is pls?

Reply:
Ah. Good catch. Actually it's not needed. I changed to validate some tests and forgot to revert it. Reverted now.